packages/app/src/components/feature/screens/mypage/MyInfoSection/index.tsx

```
@@ -0,0 +1,138 @@
+  align-items: center;
+  padding: 0 16px;
+`;
+
+const ProfileImagePlaceholder = styled.View`
+  width: 96px;
+  height: 96px;
+  border-radius: 48px;
+  background-color: ${COLORS.gray300};
+`;
+
+const ProfileChangeButton = styled.TouchableOpacity``;
```

ESLint

[error] 'ProfileChangeButton' is assigned a value but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)